gamedata/cbasenpc.txt

```diff
@@ -182,6 +238,12 @@
 				"windows"		"\x55\x8B\xEC\x83\xEC\x14\x53\x56\x8B\xF1\x57\x8D\x2A\x2A\xC7\x06"
 				"linux"			"@_ZN11CBaseEntityC1Eb"
 			}
+			"CBaseEntity::PhysicsMarkEntitiesAsTouching"
```

Comment:
Since this is a long signature (and more likely to change than others), can you add instructions on how to obtain this signature quickly in the future?

Reply:
I could have sworn I put the instructions, will fix right away.